language/vm/vm-runtime/src/data_cache.rs

```diff
@@ -80,6 +80,22 @@ impl<'block> RemoteCache for BlockDataCache<'block> {
     }
 }
 
+pub(crate) struct RemoteStorage<'a>(&'a dyn StateView);
```

Comment:
@sunshowers is there a better way to do this other than creating a wrapper struct?

Reply:
Can you do `impl<'a> RemoteCache for &'a dyn StateView`?

Looks good otherwise!

Reply:
`   = note: expected type `&dyn data_cache::RemoteCache`
              found type `&dyn libra_state_view::StateView``
I got some error messages like this :(

Reply:
ah ok, want to put in a comment about this?